src/stores/CountryStore.js

```diff
@@ -0,0 +1,37 @@
+//  ### 🤠 ###
+import { defineStore } from 'pinia'
+import { vdbFetchData } from '@/utils/api'
+
+export const useCountryStore = defineStore('CountryStore', {
+  state: () => {
+    return {
+      fetching: false,
+      token: JSON.parse(localStorage.getItem('user-store')).token,
+      countrysPage: null,
+      countrysArray: []
+    }
+  },
+  actions: {
+    async getCountrys() {
+      // If there's no token, something went wrong
+      if (!this.token) throw Error('VoloDB-ERROR\n🙅‍♀️ ups! not logged in.')
+      try {
+        this.countrysPage = await vdbFetchData(`/countrys`, 'GET', this.token)
+        this.createCountrysArray()
```

Comment:
you don't need createCountrysArray... becaus you already got it from the backend (-> no pagination)